src/dd_send.c

```diff
@@ -188,7 +190,7 @@ int DDR_finalize(DDReq *req) {
     return DDRC_EINVAL;
   //  as_sprintf(req->as_header, "POST http://%s%s HTTP/1.1\r\n", req->host,
   //             "/v1/input");
-  as_sprintf(req->as_header, "POST %s HTTP/1.1\r\n", "/v1/input");
+  as_sprintf(req->as_header, "POST %s HTTP/1.1\r\n", "/profiling/v1/input");
```

Comment:
David, if this needs to be conditional let's sort it out in a separate PR. Thanks!

Reply:
Ah, yeah--this is going to break intake for demo applications depending on direct backend access, but that's not a big deal.  Looks good.